dev/dll/Microsoft.UI.Xaml.vcxproj

```diff
@@ -180,7 +180,7 @@
     <Link>
       <ModuleDefinitionFile>Microsoft.UI.Xaml.def</ModuleDefinitionFile>
       <GenerateDebugInformation Condition="'$(Configuration)'=='Debug'">$(GenerateDebugInformation)</GenerateDebugInformation>
-      <AdditionalDependencies>dxguid.lib;%(AdditionalDependencies)</AdditionalDependencies>
+      <AdditionalDependencies>advapi32.lib;dxguid.lib;%(AdditionalDependencies)</AdditionalDependencies>
```

Comment:
>advapi32.lib [](start = 30, length = 12)

I don't think this lib can load on OneCore-based SKUs.  You should probably use an apiset like api-ms-win-core-registry-l1-1-0.

Make sure to try this MUXC out on a OneCore SKU (like xbox) to be sure this doesn't prevent it from loading entirely.

Reply:
Now linking with MinCore.lib  (The https://docs.microsoft.com/en-us/windows/desktop/apiindex/windows-8-api-sets webpage was useful too).
I created a OneCore Pusheen VM and will use it to test in a OneCore environment.